src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,54 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type MovieCardProps = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<MovieCardProps> = ({ movie }) => {
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={movie.imgUrl}
+            alt={movie.title}
+          />
```

Comment:
```suggestion
            data-cy="MovieImage"
            src={imgUrl}
            alt={title}
          />
```

Use destructuring for getting access to values of props object

Reply:
thank you, fixed